### ISSUE TITLE
bug: installAfter or installsAfter?

### ISSUE THREAD

**Issue by yf-yang** [devcontainers/spec#160]
I see the [feature doc](https://containers.dev/implementors/features/) and [devcontainer.json file doc](https://containers.dev/implementors/json_reference/) says the name should be `installsAfter`, but `installAfter` appears once [here](https://containers.dev/implementors/features/#:~:text=based%20on%20the-,installAfter,-property%20used%20by).

It seems there is already lots of confusion, because in `devcontainers/features` they are using `installsAfter` (See the search result [here](https://github.com/devcontainers/features/search?q=installsafter)), but in `devcontainers/features` people are using `installAfter` (See the search result [here](https://github.com/devcontainers-contrib/features/search?q=installafter)).

I guess the template is wrong.

**Comment by joshspicer** [comment]
yes, that one instance of `installAfter` is a typo, the correct property name is `installsAfter`
